website/cue/reference/releases/0.45.0.cue

```diff
@@ -22,9 +22,8 @@ releases: "0.45.0": {
 		"""
 
 	known_issues: [
-		"""
-			To prevent a crash at startup, avoid name clashes between enrichment table names and other components like sources, transforms, and sinks in the configuration. To resolve this, you can rename the enrichment table to a unique name that isn't already used by any other component.
-			""",
+		"To prevent a crash at startup, avoid name clashes between enrichment table names and other components like sources, transforms, and sinks in the configuration. To resolve this, you can rename the enrichment table to a unique name that isn't already used by any other component.",
```

Comment:
```suggestion
		"To prevent a crash at startup, avoid name conflicts between enrichment table names and other components like sources, transforms, and sinks in the configuration. To resolve a conflict, rename the enrichment table to a unique name that isn't already being used by another component.",
```